File: code/Java/interpreter/expr/ConstBoolExpr.java
package interpreter.expr;

public class ConstBoolExpr extends BoolExpr {

    private boolean value;

    public ConstBoolExpr(int line, boolean value) {
        super(line);
        this.value = value;
    }

    public boolean expr() {
        return value;
    }
    
}

File: code/C++/interpreter/expr/ConstBoolExpr.cpp
#include "ConstBoolExpr.h"

ConstBoolExpr::ConstBoolExpr(int line, bool value)
	: BoolExpr(line), m_value(value) {
}

ConstBoolExpr::~ConstBoolExpr() {
}

bool ConstBoolExpr::expr() {
	return m_value;
}
